cns/cnsclient/cnsclient.go

```diff
@@ -54,7 +52,9 @@ func GetCnsClient() (*CNSClient, error) {
 	if cnsClient == nil {
 		err = &CNSClientError{
 			types.UnexpectedError,
-			fmt.Errorf("[Azure CNSClient] CNS Client not initialized")}
+			//nolint:goerr113
```

Comment:
reason for this annotation?

Reply:
the goerr113 linter complains about returning dynamic errors, like the one in the next line: `fmt.Errorf(...)`, but I don't want to include fixing those in this PR scope since I didn't actually add/change them. `//nolint` ignores them for now.
I have a backlog item for a big errors cleanup to fix this and unchecked errors globally.